spec/spec_helpers/fixture_helper.rb

```diff
@@ -22,8 +22,15 @@ def config_defaults
     }
   end
 
-  def make_page(data = {})
-    Jekyll::Page.new(site, config_defaults["source"], "", "page.md").tap { |page| page.data = data }
+  def make_page(data = {}, pager = nil)
+    data["paginated"] = true unless pager == nil
+    page = Jekyll::Page.new(site, config_defaults["source"], "", "page.md").tap { |page| page.data = data }
```

Comment:
FYI, `.tap` is used in lieu of assigning a local variable.
Using both is just smelly code.